Failed to compile rust code

Looks like the toolchain used by default is the stable one and bitwarden_rs requires the nightly toolchain.

```
2020-12-30 15:23:17,206: WARNING -    Compiling devise_core v0.3.0 (https://github.com/SergioBenitez/Devise.git?rev=e58b3ac9a#e58b3ac9)
2020-12-30 15:23:18,112: WARNING - error[E0554]: `#![feature]` may not be used on the stable release channel
2020-12-30 15:23:18,117: WARNING -  --> /var/www/bitwarden/.cargo/git/checkouts/devise-934be3a303f7c6d6/e58b3ac/core/src/lib.rs:1:1
2020-12-30 15:23:18,120: WARNING -   |
2020-12-30 15:23:18,123: WARNING - 1 | #![feature(proc_macro_diagnostic, proc_macro_span)]
2020-12-30 15:23:18,138: WARNING -   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2020-12-30 15:23:18,140: WARNING - 
2020-12-30 15:23:18,143: WARNING - error[E0554]: `#![feature]` may not be used on the stable release channel
2020-12-30 15:23:18,145: WARNING -  --> /var/www/bitwarden/.cargo/git/checkouts/devise-934be3a303f7c6d6/e58b3ac/core/src/lib.rs:2:1
2020-12-30 15:23:18,147: WARNING -   |
2020-12-30 15:23:18,150: WARNING - 2 | #![feature(crate_visibility_modifier)]
2020-12-30 15:23:18,152: WARNING -   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2020-12-30 15:23:18,154: WARNING - 
2020-12-30 15:23:18,157: WARNING - error[E0554]: `#![feature]` may not be used on the stable release channel
2020-12-30 15:23:18,159: WARNING -  --> /var/www/bitwarden/.cargo/git/checkouts/devise-934be3a303f7c6d6/e58b3ac/core/src/lib.rs:3:1
2020-12-30 15:23:18,161: WARNING -   |
2020-12-30 15:23:18,163: WARNING - 3 | #![feature(concat_idents)]
2020-12-30 15:23:18,165: WARNING -   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
2020-12-30 15:23:18,168: WARNING - 
2020-12-30 15:23:19,174: WARNING -    Compiling syn-mid v0.5.0
2020-12-30 15:23:19,579: WARNING - error: aborting due to 3 previous errors
2020-12-30 15:23:19,581: WARNING - 
2020-12-30 15:23:19,583: WARNING - For more information about this error, try `rustc --explain E0554`.
2020-12-30 15:23:19,586: WARNING - error: could not compile `devise_core`.
2020-12-30 15:23:19,589: WARNING - warning: build failed, waiting for other jobs to finish...
2020-12-30 15:24:33,768: WARNING - error: build failed
```

Reading the install script, I find this : https://github.com/YunoHost-Apps/bitwarden_ynh/blob/e2e958b1885f7a08e0d5afe5dada5c0dd44dd671/scripts/install#L125

That would mean that if cargo is installed system-wide, it will use it instead of the one installed locally in `$final_path`.

--- a/scripts/install
+++ b/scripts/install
@@ -119,7 +119,7 @@ chown -R "$app":"$app" "$final_path"
 
 # Install rustup with the toolchain needed by bitwarden_rs
 pushd "$final_path"
-	sudo -u "$app" RUSTUP_HOME="$final_path"/.rustup CARGO_HOME="$final_path"/.cargo bash -c 'curl -sSf -L https://static.rust-lang.org/rustup.sh | sh -s -- -y --default-toolchain=$(cat build/rust-toolchain)'
+	sudo -u "$app" RUSTUP_HOME="$final_path"/.rustup CARGO_HOME="$final_path"/.cargo bash -c 'curl -sSf -L https://static.rust-lang.org/rustup.sh | sh -s -- -y --default-toolchain nightly'
 popd
 
 export PATH="$PATH:$final_path/.cargo/bin:$final_path/.local/bin:/usr/local/sbin" 

--- a/scripts/upgrade
+++ b/scripts/upgrade
@@ -109,7 +109,7 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	# Install rustup with the toolchain needed by bitwarden_rs
 	pushd "$final_path"
-		sudo -u "$app" RUSTUP_HOME="$final_path"/.rustup CARGO_HOME="$final_path"/.cargo bash -c 'curl -sSf -L https://static.rust-lang.org/rustup.sh | sh -s -- -y --default-toolchain=$(cat build/rust-toolchain)'
+		sudo -u "$app" RUSTUP_HOME="$final_path"/.rustup CARGO_HOME="$final_path"/.cargo bash -c 'curl -sSf -L https://static.rust-lang.org/rustup.sh | sh -s -- -y --default-toolchain nightly'
 	popd
 
 	export PATH="$PATH:$final_path/.cargo/bin:$final_path/.local/bin:/usr/local/sbin" 

